File: Python/src/gestorAplicacion/usuario/Equipaje.py
from logistica import *
class Equipaje(CargaExtra):
    equipajes=[]
    
    def __init__(self,precio,tamaño):
        super(precio, tamaño)
        Equipaje.equipajes.append(self)
    
    def agregar(self):
        Equipaje.equipajes.append(self)
    
    @classmethod
    def getEquipajes(cls):
        return cls.equipajes
    @classmethod
    def setEquipajes(cls,equipaje):
        cls.equipajes = equipaje

File: src/gestorAplicacion/usuario/Equipaje.java
package gestorAplicacion.usuario;
import java.util.*;
import gestorAplicacion.logistica.*;

public class Equipaje extends CargaExtra {

    private static ArrayList<Equipaje> equipajes = new ArrayList<Equipaje>();

    //constructores
    public Equipaje(int precio, String tamaño) {
        super(precio, tamaño);
        agregar();
    }

    @Override
    public void agregar() {
        equipajes.add(this);
        
    }

    //getter and setter
    public static ArrayList<Equipaje> getEquipajes() {
        return equipajes;
    }
    public static void setEquipajes(ArrayList<Equipaje> equipaje) {
        Equipaje.equipajes = equipaje;
    }
}
